direct.mk

```diff
@@ -0,0 +1,156 @@
+.DEFAULT_GOAL = all
+.PHONY: all
+all: check binaries test integration-tests ## run fmt, vet, lint, build the binaries and run the tests
+
+.PHONY: check
+check: fmt vet lint ineffassign misspell
+
+.PHONY: ci
+ci: check binaries checkprotos coverage coverage-integration ## to be used by the CI
+
+.PHONY: AUTHORS
+AUTHORS: .mailmap .git/HEAD
+	git log --format='%aN <%aE>' | sort -fu > $@
+
+# This only needs to be generated by hand when cutting full releases.
+version/version.go:
+	./version/version.sh > $@
+
+.PHONY: setup
+setup: ## install dependencies
+	@echo "🐳 $@"
+	# TODO(stevvooe): Install these from the vendor directory
+	@go get -u github.com/golang/lint/golint
+	#@go get -u github.com/kisielk/errcheck
+	@go get -u github.com/gordonklaus/ineffassign
+	@go get -u github.com/client9/misspell/cmd/misspell
+	@go get -u github.com/lk4d4/vndr
+	@go get -u github.com/stevvooe/protobuild
```

Comment:
nit: it may be handy, if we're going to use `docker-sync`, to include installation in the makefile, either here or in the other Makefile. Not necessarily as part of the `setup` target, if you don't think that's appropriate.

this is not a blocker, and if you decline to make this change, i'm unopposed to merging as-is.

Reply:
We run `make setup` in the Dockerfile, so it's already baked into the image

Reply:
Oooh. Misunderstood you. It sure does. It comes as a gem. The idea here is that people who want to use it need to install it beforehand though. I don't want to go down the rabbithole of rvm and chruby and what not...